models/staging/stg_dbt__exposures.sql

```diff
@@ -3,6 +3,10 @@ with base as (
     select *
     from {{ source('dbt_artifacts', 'exposures') }}
 
+    {% if is_incremental() %}
+      -- this filter will only be applied on an incremental run
+      where run_started_at > (select IFNULL(max(run_started_at), TIMESTAMP("1900-01-01")) from {{ this }})
+    {% endif %}
```

Comment:
```suggestion

```

Please could you remove all these if is_incremental blocks? All models are views.

Reply:
I will put it in a separate pull request as it might be relevant to use incrementals for staging tables. But I agree it should not be part of this PR.

Will do it over the weekend

Reply:
@NiallRees : removed from the PR